transactions.md

```diff
@@ -101,7 +102,7 @@ path, only spendable after `X` blocks.
     - value: `<unvault_tx outputs[0] value - tx_fee - 330>`
```

Comment:
Shouldn't we also adjust the output[0] value to: `<unvault_tx outputs[0] value - tx_fee - (2*32* tx_size_vbytes)>`